gpustat.py

```diff
@@ -405,14 +410,18 @@ def print_gpustat(json=False, **args):
     '''
     Display the GPU query results into standard output.
     '''
+    gpu_stats = GPUStatCollection.new_query()
     try:
         gpu_stats = GPUStatCollection.new_query()
     except CalledProcessError:
         sys.stderr.write('Error on calling nvidia-smi\n')
         sys.exit(1)
 
     if json:
-        gpu_stats.print_json(sys.stdout)
+        if args['ret_json']:
+            return gpu_stats.print_json(None)
```

Comment:
Although this line would return the JSON-like object as dict, it doesn't match the name and looks confusing a bit.